Google.Api.Generator.IntegrationTests/ComplianceRestTest.cs

```diff
@@ -0,0 +1,101 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Api.Gax.Grpc;
+using Google.Protobuf;
+using Google.Showcase.V1Beta1;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Google.Api.Generator.IntegrationTests
+{
+    /// <summary>
+    /// Compliance test for the REST transport.
+    /// </summary>
+    public class ComplianceRestTest : ShowcaseTestBase<ComplianceClient, ComplianceClientBuilder>
+    {
+        private static ComplianceSuite s_testFile = TestResources.ParseJson<ComplianceSuite>("compliance_suite.json");
+        public static IEnumerable<object[]> Tests =>
+            from grp in s_testFile.Group
+            from rpc in grp.Rpcs
+            from req in grp.Requests
+            select new object[] { new SerializableTest(grp.Name, rpc, req) };
+
+        [SkippableTheory]
+        [InlineData(false)]
+        [InlineData(true)]
+        public void GetEnum(bool unknownEnum)
```

Comment:
nit: I think there's something like:
```suggestion
        public void GetEnum([Combinatorial] bool unknownEnum)
```

Reply:
That's definitely an option if we also reference the relevant NuGet package - but I personally find the inline data simpler to understand anyway :)